File: solutions/3408-count-the-number-of-special-characters-i/solution.py
class Solution(object):
    def numberOfSpecialChars(self, word):
        """
        :type word: str
        :rtype: int
        """
        c=0
        for i in set(word):
            if i.islower():
                if i.upper() in word:
                    c+=1
            else:
                if i.lower() in word:
                    c+=1
        return c/2

File: solutions/3408-count-the-number-of-special-characters-i/solution.java
class Solution {
    public int numberOfSpecialChars(String word) {
        Set<Character>ls=new HashSet<>();
        int count=0;
        for(char a:word.toCharArray())
            ls.add(a);
        for(char a:ls)
            if(word.indexOf(a-32)!=-1)
                count++;
        return count;
    }
}
